.tekton/cnf-tests-pull-request.yaml

```
@@ -0,0 +1,620 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/openshift-kni/cnf-features-deploy?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
+      == "master"
+  creationTimestamp: null
+  labels:
+    appstudio.openshift.io/application: cnf-tests-4-19
+    appstudio.openshift.io/component: cnf-tests-4-19
+    pipelines.appstudio.openshift.io/type: build
+  name: cnf-tests-4-19-on-pull-request
+  namespace: telco-5g-tenant
+spec:
+  params:
+    - name: git-url
+      value: '{{source_url}}'
+    - name: revision
+      value: '{{revision}}'
+    - name: output-image
+      value: quay.io/redhat-user-workloads/telco-5g-tenant/cnf-tests-4-19:on-pr-{{revision}}
+    - name: image-expires-after
+      value: 5d
+    - name: dockerfile
+      value: cnf-tests/.konflux/Dockerfile
+    - name: prefetch-input
+      value: '{"type": "rpm", "path": "cnf-tests/.konflux"}'
```

I think an explicit hermetic setting must be set to ensure network isolation:

```suggestion
      value: '{"type": "rpm", "path": "cnf-tests/.konflux"}'
    - name: hermetic
      value: "true"
```

See https://konflux.pages.redhat.com/docs/users/how-tos/configuring/hermetic-builds.html

---

will conforma issue an error if  hermetic is not explicitly set here ?

---

I'm not sure. Hermetic builds are strongly recommended but not mandatory from what I read.

---

I believe conforma will fail if hermetic is not enabled

---

Thanks for highlighting Carlos. I will fix it. I wonder how come this was accepted so far by the CI without this param, perhaps they newly added it.